### ISSUE TITLE
Complex timestepping

### ISSUE THREAD

**Issue by jbrezmorf** [flow123d/flow123d#334]
Allow prescribe different time steps during different time periods.

prerequisite: #489 

Apply TimeTable class for prescribing max_dt and min_dt in TimeGovernor.

Open questions:
- in order to prescribe min_dt and max_dt in one timeline we need to make them a tuple
- how to allow setting just max_dt then

**Comment by jbrezmorf** [comment]
Depends on #132.

**Comment by jbrezmorf** [comment]
prerequisite for #694

**Comment by jbrezmorf** [comment]
Duplicate of #926

**Comment by jbrezmorf** [comment]
Duplicate of solved issue.
